client/src/pages/ManageBookings/Booking.tsx

```diff
@@ -0,0 +1,47 @@
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Avatar from '@material-ui/core/Avatar';
+import Typography from '@material-ui/core/Typography';
+import SettingsIcon from '@material-ui/icons/Settings';
+
+import useStyles from './useStyles';
+
+interface BookingProps {
+  outlined?: boolean;
+  accepted?: boolean;
+  large?: boolean;
+  date?: Date;
+  name?: string;
+  avatar?: string;
+}
+
+const Booking: React.FC<BookingProps> = ({ outlined, accepted, large, date, name, avatar }) => {
+  const classes = useStyles();
+
+  const dateString = date && date.toString().split('T')[0];

```

Comment:
This doesn't seem like a good way of doing this, what exactly is the end result here?

Reply:
This results in a date without the time attached. Is there a better way to do this?